vtex/omnishipping
# Shipping Manager

#### What is the purpose of this pull request?
The time has come. This PR merges the Shipping Manager refactor branch into master and prepares for launch.

All commits (except for release and changelog commits) in the `refactor/shipping-manager` branch were already reviewed in [past PRs](https://github.com/vtex/omnishipping/pulls?q=is%3Apr+base%3Arefactor%2Fshipping-manager). There's no need to review all changes except for the changelog.

#### How should this be manually tested?
Use these workspaces ([lean](https://shpmgr2--vtexgame1.myvtex.com/checkout/cart/add?sku=289&qty=1&seller=1)/[nolean](https://shpmgr2--vtexgame1nolean.myvtex.com/checkout/cart/add?sku=289&qty=1&seller=1)), which have ***just omnishipping*** linked. I can't provide a specific test plan, so just test it with random items and see if everything is fine.

#### Types of changes
